site/layouts/partials/home/masthead.html

```diff
@@ -17,7 +17,9 @@ <h1 class="mb-3">Build fast, responsive sites with Bootstrap</h1>
         <p class="text-muted mb-0">
           Currently <strong>v{{ .Site.Params.current_version }}</strong>
           <span class="px-1">&middot;</span>
-          <a href="/docs/versions/" class="link-secondary">Previous releases</a>
+          <a href="https://getbootstrap.com/docs/4.6/getting-started/introduction/" class="link-secondary">Jump to v4.x</a>
```

Comment:
I'm thinking maybe this should be automated too? I mean, it's another place we need to manually update apart from the version toggler.

But I'm not sure how to automate  it yet.